libs/cases/frontend/view/src/lib/components/detail-case/detail-case.component.ts

```diff
@@ -27,5 +30,135 @@ export class DetailCaseComponent {
         return _case.find((_case) => String(_case.id) === this.route.snapshot.params['id']);
     });
 
-    constructor(private route: ActivatedRoute) {}
+    isEditing = false;
+    wasValidated = false;
+    datePattern = /^\d{4}-\d{2}-\d{2}T00:00:00\.000Z$/;
+
+
+    constructor(private route: ActivatedRoute, private readonly _modalService: ModalService, private readonly toastService: ToastService) {
+    }
+
+    deleteCase() {
+        const caseId = this.mapCaseId(this.casedetail());
+        if (caseId !== undefined) {
+            this._browseFacade.deleteCase(caseId).subscribe();
```

Comment:
remove unnecessary subscribe

Reply:
This subscribe is actually necessary, otherwise it will not perform any type of backend call. The same goes for the subscribe in the update method.